diff --color -u a/config.def.h b/config.def.h
--- a/config.def.h	2022-01-10 18:41:59.439035164 +0300
+++ b/config.def.h	2022-01-10 18:44:18.611025434 +0300
@@ -74,7 +74,7 @@
 static int bellvolume = 0;
 
 /* default TERM value */
-char *termname = "st-256color";
+char *termname = "xterm-256color";
 
 /*
  * spaces per tab
@@ -96,32 +96,32 @@
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    "#767676",
+    "#f2201f",
+    "#23fd00",
+    "#fffd00",
+    "#1a8fff",
+    "#fd28ff",
+    "#14ffff",
+    "#ffffff",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+    "#000000",
+    "#cc0403",
+    "#19cb00",
+    "#cecb00",
+    "#0d73cc",
+    "#cb1ed1",
+    "#0dcdcd",
+    "#dddddd",
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
 	"#cccccc",
 	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+	"#ffffff", /* default foreground colour */
+	"#202020", /* default background colour */
 };
 
 
