diff --color -u a/config.def.h b/config.def.h
--- a/config.def.h	2022-01-10 18:47:41.097011278 +0300
+++ b/config.def.h	2022-01-10 18:48:22.985008349 +0300
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Fira Code:pixelsize=18:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
