Только в b: .git
diff --color -u a/st.c b/st.c
--- a/st.c	2022-01-11 16:40:33.467336704 +0300
+++ b/st.c	2022-01-11 16:40:20.337335771 +0300
@@ -1174,18 +1174,9 @@
 	if (sel.ob.x == -1)
 		return;
 
-	if (BETWEEN(sel.nb.y, orig, term.bot) != BETWEEN(sel.ne.y, orig, term.bot)) {
-		selclear();
-	} else if (BETWEEN(sel.nb.y, orig, term.bot)) {
-		sel.ob.y += n;
-		sel.oe.y += n;
-		if (sel.ob.y < term.top || sel.ob.y > term.bot ||
-		    sel.oe.y < term.top || sel.oe.y > term.bot) {
-			selclear();
-		} else {
-			selnormalize();
-		}
-	}
+	sel.ob.y += n;
+	sel.oe.y += n;
+	selnormalize();
 }
 
 void
diff --color -u a/x.c b/x.c
--- a/x.c	2022-01-11 16:40:33.467336704 +0300
+++ b/x.c	2022-01-11 16:39:50.803333670 +0300
@@ -723,6 +723,12 @@
 		return;
 	}
 
+    Arg arg = { .i = 1 };
+    if (e->xbutton.y > win.h)
+        kscrolldown(&arg);
+    if (e->xbutton.y < 0.f)
+        kscrollup(&arg);
+
 	mousesel(e, 0);
 }
 
